engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/openapi/StandaloneOpenApiGenerator.scala

```diff
@@ -0,0 +1,76 @@
+package pl.touk.nussknacker.engine.standalone.openapi
+
+import io.circe.Json
+import io.circe.syntax._
+import io.circe.generic.auto._
+import pl.touk.nussknacker.engine.standalone.StandaloneProcessInterpreter
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+object StandaloneOpenApiGenerator {
```

Comment:
We agreed not to use external library since model we produce is very simple and it seems that it is easier to manage/change it manually. I am open for discussion/suggestions if any